### ISSUE TITLE
Root

### ISSUE THREAD

**Issue by Unknownhuman09** [mkdirlove/ANTI-PISO-WIFI#7]
Need sya ng root sayang

**Comment by revdeluxe** [comment]
Obviously, It needs elevated permissions to change your device's MAC ADDRESS. Para hindi malaman ng PISO wifi server na hindi fake itong user na to.

Some Piso Wifi uses accounts to avoid this kind of exploit. But of course there's still a way to get to the server's database to manipulate data. Of course by also using a part of this Source Code.

@mkdirlove not an issue, issue described is too broad to fix for a small project. 
> Sorry for the ping.

**Comment by joshuadelossantos2** [comment]
> Need sya ng root sayang

Kaya Pala no su binary found
